src/Config/SettingsTab.ts

```diff
@@ -291,6 +291,68 @@ export class SettingsTab extends PluginSettingTab {
                     await this.plugin.saveSettings();
                 });
             });
+
+        // ---------------------------------------------------------------------------
+        containerEl.createEl('h4', { text: 'Reminder Settings' });
+        // ---------------------------------------------------------------------------
+
+        new Setting(containerEl)
+            .setName('Reminder Format')
+            .setDesc(
+                SettingsTab.createFragmentWithHTML(
+                    '<p class="tasks-setting-important">Do not change if reminders already exist, otherwise you will not be notified unless manually converted to new format.</p>' +
+                        '<p>The format that Tasks uses to read and write reminders time i.e 12hr 6:00 pm or 24hr 18:00.</p>',
+                ),
+            )
+            .addDropdown((dropdown) => {
+                let k: keyof typeof TIME_FORMATS;
+                for (k in TIME_FORMATS) {
+                    dropdown.addOption(TIME_FORMATS[k], k);
+                }
+                const settings = getSettings().reminderSettings;
+
+                dropdown.setValue(settings.dateTimeFormat).onChange(async (value) => {
+                    settings.dateTimeFormat = value;
+                    updateSettings({ reminderSettings: settings });
+                    await this.plugin.saveSettings();
+                });
+            });
+
+        new Setting(containerEl)
+            .setName('Daily Reminder Time')
+            .setDesc(
+                SettingsTab.createFragmentWithHTML(
+                    '<p>When daily reminders should be triggered. Should be in same format as above i.e 12 or 24hr.</p>',
```

Comment:
There is no feedback to the user when the formats differ.

I feel that, at a minimum, when the user changes the format selection above, this display text should be updated.

Reply:
I haven’t followed your changes over the last day (many thanks for them) but if the parsing is more flexible now, does the format really need to match still, or can this wording just be relaxed instead?